index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Hussen Portfolio</title>
+  </head>
+  <body>
+    <!-- The toolbar (or header) section -->
+
+    <header class="topBar-nav">
+      <div class="logo">
+        <a href="#"><img src="img/My-Logo.png" alt="HY" /></a>
+      </div>
+      <div class="icon-Union">
+        <a href="#"><img src="img/Union.png" alt="Three Dashs" /></a>
+      </div>
+    </header>
+
+    <main id="main">
+      <!-- The headline section (right after the header) -->
+      <section class="content-section">
+        <div class="main-Container">
+          <h1 class="headingTxt">I’m Hussen. Glad to see you!</h1>
+          <div class="text-2">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+
+          <div class="lets-connect">
+            <a href="#">LET’S CONNECT</a>
+          </div>
+          <ul class="social-media-icons">
+            <li class="list">
+              <a href="https://twitter.com/">
+                <img src="img/t.png" alt="tiwetter" />
```

Comment:
- [ ] Kindly reduce the font-size of this social media icon. It's kind of bigger than others.

### Your Design
![your-design](https://user-images.githubusercontent.com/26597432/234683508-a397cef3-5208-4de6-8637-a6c6b0b4ddba.png)

### Figma Design
![temp-design](https://user-images.githubusercontent.com/26597432/234683546-ed7612cf-f2c8-4c0b-a181-660a191439e9.png)